bgpd/bgp_vty.c

```diff
@@ -9654,11 +9634,31 @@ static void bgp_show_neighbor_graceful_restart_capability_per_afi_safi(
 				} else {
 					vty_out(vty, "No\n");
 					vty_out(vty,
-						"     EoRSentAfterUpdate   : ");
+						"      End-of-RIB send after update: ");
```

Comment:
Just one instance of Send. Otherwise code looks good.

Reply:
ups, fixed.